components/collective-page/sections/Contributions.js

```diff
@@ -127,6 +127,20 @@ const contributionsSectionQuery = gqlV2/* GraphQL */ `
       settings
       type
       isHost # TODO: Fetch number of hosted collectives
+      imageUrl(height: 128)
+      backgroundImageUrl(height: 200)
+      ... on Event {
+        parent {
```

Comment:
Maybe it's time to decide on using` parent` vs `parentAccount`, https://github.com/opencollective/opencollective/issues/4549 Should we go for `parent`?

Reply:
Shared some feedback in https://github.com/opencollective/opencollective/issues/4549